### PR TITLE
Update day2_finalproject_tipcalculator.py

### PR DESCRIPTION
**Pull request by nordiCode**
Added some bounds to the tip

### CODE REVIEW

File: day2_finalproject_tipcalculator.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 print("Welcome to the tip calculator.")
 
 total_bill = input("What was the total bill? ")
 tip_amount = input("How much would you like to tip? 10, 12, 15 or 18? ")
+
+
+#some bounds on tipping 
+if(int(tip_amount)<0):
+    print("Come on you have to tip something")
+elif(int(tip_amount)>100):
```
Comment (KootenayCode) [inline day2_finalproject_tipcalculator.py]: What if:
elif(int(tip_amount)>total_bill)

Comment (nordiCode) [reply day2_finalproject_tipcalculator.py]: I can do this!

**Review by KootenayCode — CHANGES_REQUESTED** [review]
Take a look at line 11, rest looks great. Thanks!

**Review by KootenayCode — APPROVED** [review]

**Comment by KootenayCode** [comment]
Testing out some github features

**Pull request merged**